src/api/config/clock.rb

```diff
@@ -27,6 +27,10 @@ module Clockwork
     SendEventEmailsJob.perform_later
   end
 
+  every(30.seconds, 'instrument delayed job queue') do
```

Comment:
Not sure 30 seconds is the right frequency...

Reply:
I would say it's fine. You do not fancy supergreat granuality - but rather trends for alarms. 

But I would make sure it's not trying to report anything to a not configured influxdb. a9ac08cec02481f89958ea74f6007076082118bf uses CONFIG['influxdb_hosts'] for that

Reply:
I think this is taken care of in `InfluxDB::Rails`, let me check.

Reply:
See now...